include/oneapi/dpl/internal/random_impl/exponential_distribution.h

```diff
@@ -0,0 +1,195 @@
+// -*- C++ -*-
+//===-- exponential_distribution.h ----------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Public header file provides implementation for Exponential Distribution
+
+#ifndef DPSTD_EXPONENTIAL_DISTRIBUTION

```

Comment:
It is an old style. Let's change it to _ONEDPL_EXPONENTIAL_DISTRIBUTION

Reply:
done